test/babashka/bbin/scripts/maven_jar_test.clj

```diff
@@ -28,18 +29,19 @@
       (is (= `{~'http-server ~maven-lib} (tu/run-ls))))))
 
 (def upgraded-lib
-  (assoc-in maven-lib [:coords :mvn/version] "0.1.12"))
+  (assoc-in maven-lib [:coords :mvn/version] "0.1.13"))
```

Comment:
This seems to be a failing test on master too - `0.1.13` is the latest `org.babashka/http-server` version.